packages/flutter/lib/src/widgets/route_notification_messages.dart

```diff
@@ -9,33 +9,29 @@ import 'navigator.dart';
 
```

Comment:
The analyzer is complaining that the above import (`import 'navigator.dart;`) isn't used.